scripts/EN/Harry

```diff
@@ -3498,9 +3498,9 @@ CleanScreen(); // 0x43dc: 00000064
 LoadBackground(0x54); // 0x43e0: 00001069
 DrawBackground(); // 0x43e6: 00000065
 PlayMusic(0x6e, 0x1); // 0x43ea: 0000206c
-ShowText((4056)"I...get out the pet bottle and use a fragment of the bottom section of one of the broken bottles to pour the blue liquid into the it."); // 0x43f2: 0000100c
+ShowText((4056)"I…get out the plastic bottle and use a fragment of the bottom section of one of the broken vials to pour the blue liquid into the it."); // 0x43f2: 0000100c
 HandleInput(); // 0x43f8: 00000010
-ShowText((4057)"The sticky liquid drips into the pet bottle. The last drop lingered, but once it finally hit the surface of the liquid already inside the pet bottle I tightly closed the cap and then left the Director's Office."); // 0x43fc: 0000100c
+ShowText((4057)"The sticky liquid drips into the plastic bottle.¥nW2The last drop lingered, W1but once it finally hit the surface of the liquid already inside the plastic bottle W1I tightly closed the cap and then left the Director's Office."); // 0x43fc: 0000100c
```

Comment:
Changed pet to plastic to match the PS1 version. (PET refers to the type of plastic and is how they refer to these bottles in Japan.)